agent-eval/.env.example

```
@@ -0,0 +1,22 @@
+
+# For direct OpenAI Codex API:
+# OPENAI_API_KEY=sk-proj-...
+
+# Sandbox access
+# agent-eval uses Vercel Sandbox when Vercel access-token credentials are set.
+# Without VERCEL_TOKEN, sandbox: 'auto' falls back to Docker.
+
+# Option 1: Vercel access-token credentials (recommended for CI)
+# Create a classic personal token at: https://vercel.com/account/tokens
+VERCEL_TOKEN=your-vercel-token
+VERCEL_TEAM_ID=team_xxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VERCEL_TEAM_ID key should go before the VERCEL_TOKEN key

(UnorderedKey)